Procfile

```diff
@@ -0,0 +1 @@
+client: ./node_modules/mastarm/bin/mastarm build --serve --proxy http://localhost:4000/api
```

Comment:
I think it's better to just do npm start

Reply:
Done